ci/coverity_unsuccessful.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -uex
+
+rm -rf "_build.external${arch:-}"
+mkdir -p coverity
+rm -f coverity/*
```

Comment:
This is an odd idiom.  Usually one just does:
```suggestion
rm -f coverity
mkdir coverity
```